rdt/hyper_transformer.py

```diff
@@ -311,6 +332,25 @@ def _warn_update_transformers_by_sdtype(self, transformer, transformer_name):
                 'parameters instead.', FutureWarning
             )
 
+    def _remove_column_in_multi_column_fields(self, column):
+        """Update the column in tuple dict.
```

Comment:
Can we give a more in depth description here. It seems like this method remove on column from a field that is made up of multiple columns by removing it from the tuple. It also updates the transformer

Reply:
Yes done in [f01ceca](https://github.com/sdv-dev/RDT/pull/695/commits/f01ceca786505c287f030d20b7e58710deef127f)